.contrib/Parser/DATAS/02 - Outdoor Zones/01 Kalimdor/Dustwallow Marsh.lua

```diff
@@ -1637,6 +1738,9 @@ root(ROOTS.Zones, m(KALIMDOR, {
 					["timeline"] = { ADDED_4_0_3 },
 					["races"] = ALLIANCE_ONLY,
 					["groups"] = {
+						objective(1, {	-- 0/1 Wreath Laid at Hyal Monument
```

Comment:
Here as well, you used a Wreath quest item on a grave (which should also show the objectID and coordinate)